deploy/cron.js

```diff
@@ -21,6 +21,22 @@ logger.info("    Running /deploy/cron.js");
 logger.info("===================================");
 
 const scriptRootPath = `${process.env.ANALYTICS_ROOT_PATH}/deploy`
+const scriptUARootPath = `${process.env.ANALYTICS_UA_ROOT_PATH}/deploy`
+
+var api_ua_run = function() {
```

Comment:
New method to run our ua application and write responses to the api database.  We no longer need to send reports from UA to s3.